src/main/java/org/opensrp/web/exceptions/BusinessLogicException.java

```diff
@@ -0,0 +1,8 @@
+package org.opensrp.web.exceptions;
+
+public class BusinessLogicException extends Exception {
```

Comment:
Can this extend a more specific extension e.g RuntimeException, Illegal**Exception